content/docs/guides/author_develop/pwa-amp/amp-in-pwa.md

```diff
@@ -0,0 +1,101 @@
+
+Include Shadow AMP in the head of your page, like so:
+
+[sourcecode:html]
+<!-- Asynchronously load the AMP-with-Shadow-DOM runtime library. -->
+<script async src="https://cdn.ampproject.org/shadow-v0.js"></script>
+[/sourcecode]
+
+## Handle navigation in your Progressive Web App
+
+You’ll still need to implement this step manually. After all, we don’t know how you want to present links to content in your navigation concept. A number of lists? A bunch of cards?
+
+This step is up to you, but my guess is that you’ll fetch some JSON that returns ordered URLs with some metadata. In the end, you should end up with a function callback that fires when the user clicks on one of the links, and said callback should include the URL of the requested AMP page. If you have that, you’re all set for the final step.
+
+## Use the Shadow API to render a page inline
+
+Finally, when you want to display content after a user action, it's time to fetch the relevant AMP document and let Shadow AMP take over. First, implement a function to fetch the page, similar to this one:
+
+[sourcecode:javascript]
+function fetchDocument(url) {
+
+  // unfortunately fetch() does not support retrieving documents,
+  // so we have to resort to good old XMLHttpRequest.
+  var xhr = new XMLHttpRequest();
+
+  return new Promise(function(resolve, reject) {
+    xhr.open('GET', url, true);
+    xhr.responseType = 'document';
+    xhr.setRequestHeader('Accept', 'text/html');
+    xhr.onload = function() {
+      // .responseXML contains a ready-to-use Document object
+      resolve(xhr.responseXML);
+    };
+    xhr.send();
+  });
+}
+[/sourcecode]
+
+{% call callout('Important', type='caution') %}
+In order to simplify the above code example, we skipped over error handling. You should always make sure to catch and handle errors gracefully.
+{% endcall %}
+
+Now that we have our ready-to-use `Document` object, it's time to let AMP take over and render it. Get a reference to the DOM element that serves as container for the AMP document, then call `AMP.attachShadowDoc()`, like so:
+
+[sourcecode:javascript]
+// This can be any DOM element
+var container = document.getElementById('#container');
+
+// The AMP page you want to display
+var url = "https://my-domain/amp/an-article.html";
+
+// Use our fetchDocument method to get the doc
+fetchDocument(url).then(function(doc) {
+  // Let AMP take over and render the page
+  var ampedDoc = AMP.attachShadowDoc(container, doc, url);
+});
+[/sourcecode]
+
+{% call callout('Tip', type='success') %}
+Before you actually hand the document over to AMP, it's the perfect time to remove page elements that make sense when displaying the AMP page standalone, but not in embedded mode: For example, footers and headers.
+{% endcall %}
+
+And that's it! You should see the AMP page in all its glory, rendered as a child of your overall Progressive Web App.
```

Comment:
Rephrase/remove because I went looking for it and because it's a phrase not common for ESL.

"And that's it! Your AMP  page renders as a child of your overall Progressive Web App."